src/Files.App/UserControls/StatusBarControl.xaml

```diff
@@ -130,6 +138,14 @@
 				Text="{x:Bind SelectedItemsPropertiesViewModel.SelectedItemsCountString, Mode=OneWay}"
 				Visibility="{x:Bind SelectedItemsPropertiesViewModel.IsItemSelected, Mode=OneWay}" />
 
+			<!--  Divider  -->
+			<Border
+				x:Name="ItemSizeDivider"
+				Width="1"
+				Height="18"
+				x:Load="{x:Bind SelectedItemsPropertiesViewModel.ItemSizeVisibility, Mode=OneWay}"
+				Background="{ThemeResource ControlStrokeColorDefaultBrush}" />
+			<!--  Item Size  -->
```

Comment:
```suggestion

			<!--  Item Size  -->
```

A new line break please

Reply:
I want them grouped together because otherwise it's not so clear what the divider is going on.